examples/tutorials/advance_model_config/01_microbatch.ipynb

```diff
@@ -0,0 +1,250 @@
+{
```

Comment:
<p>Model still receives batches of normal size: they are splitted to microbatches inside the <code>model.train</code> method</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>